docker/docker-compose.yml

```diff
@@ -24,6 +24,19 @@ services:
       - QSERVER_ZMQ_INFO_ADDRESS=tcp://qserver:60625
     networks:
       - qserver_network
+  queue-monitor:
+    image: queue-monitor:latest
+    ports:
+      - 5901:5901
+    environment:
+      - QSERVER_ZMQ_CONTROL_ADDRESS=tcp://qserver:60615
+      - QSERVER_ZMQ_INFO_ADDRESS=tcp://qserver:60625
+      - DISPLAY=${LOCAL_DISPLAY_IP}:0
```

Comment:
I believe the `:0` display ID is already included in `$DISPLAY`...meaning it would show up twice here.

See the suggestion in README.